ingestion/src/lib.rs

```diff
@@ -53,7 +53,14 @@ pub type PgConnectionPool = bb8::Pool<PostgresConnectionManager<NoTls>>;
 
 pub type PooledPgConn<'a> = PooledConnection<'a, PostgresConnectionManager<NoTls>>;
 
-type ParamConversions = Arc<HashMap<String, (String, i32)>>;
+#[derive(Clone, Debug)]
+pub struct Param {
+    id: i32,
+    element_id: String,
+    is_scalar: bool,
```

Comment:
Perhaps it would be nice to have a brief comment providing context for each of these?